Tribler/Core/Modules/restapi/multichain_endpoint.py

```diff
@@ -313,7 +314,8 @@ def render_GET(self, request):
 
         nodes, edges = mc_community.get_graph(unhexlify(focus_node), neighbor_level)
 
-        return json.dumps({"focus_node": focus_node,
+        return json.dumps({"user_node": hexlify(mc_community.my_member.public_key),
+                           "focus_node": focus_node,
```

Comment:
As the first call to the core is always done with "self" as focus node, you could store the value returned by that call as focus node instead of adding it to each reply